lisp/core.slosh

```diff
@@ -813,6 +918,3 @@ Example:
  %#
   (defmacro substitute (lst old-item new-item & mods)
        `(nsubstitute! (to-list ~lst) ~old-item ~new-item ~@mods))
-
-(load "iterator.slosh")
```

Comment:
Not sure I love not loading iterator.slosh (test.slosh seems fine).  I guess it is not a bid deal to load it first but I kinda expect iterators to be part of core and always available, even if you have to prepend a namespace sometimes.